examples/grpc-client/README.md

```diff
@@ -48,6 +48,7 @@ Finally run the Dex client providing the CA certificate, client certificate and
 Running the gRPC client will cause the following API calls to be made to the server
 1. CreatePassword
 2. ListPasswords
+3. GetPassword
 3. DeletePassword
```

Comment:
[nit] I know it doesn't matter for presentation, but it's nicer to read this in raw form if the numbers are actually consecutive -- can you make `DeletePassword` a `4.`?

Reply:
I was trying to maintain the style, but I'm just gonna make them all `1.`